DataProc/Common/readAliEnCollection.C

```diff
@@ -0,0 +1,18 @@
+void readAliEnCollection(const char *in, const char *out) {
```

Comment:
This is the new XML parser. It is based on ROOT. We always build our ROOT with XML support, so this will work everywhere, including the Grid.